packages/core/src/types-hoist/options.ts

```diff
@@ -307,9 +307,12 @@ export interface Options<TO extends BaseTransportOptions = BaseTransportOptions>
   /**
    * If this is set to false, default integrations will not be added, otherwise this will internally be set to the
    * recommended default integrations.
+   *
+   * It is deprecated to pass `Integrations[]` here. This capability will be removed in v10.
+   *
+   * TODO(v10): Remove `Integration[]` support.
    */
   defaultIntegrations?: false | Integration[];
```

Comment:
l: not sure if you already tried this but if we split up the type and union-type it again, can we somehow add `@deprecated` to the Integration[] type? might not work, so feel free to disregard.

Reply:
I tried a few ways but could not get it to work 😬 if we do e.g. this:

```typescript
/** @deprecated oh no */
type DeprecatedStuff = Integration[];

// ...
interface {
  defaultIntegrations: false | DeprecatedStuff;
}
```

then it complains _in there_ that this is deprecated (so _we_ need to add an eslint-disable comment), but for users its fine to pass this 😬